specs/darwin/crash_logs.table

```diff
@@ -0,0 +1,27 @@
+table_name("crash_logs")
+description("Application Crash Logs")
+schema([
+    Column("pid", BIGINT, "Process (or thread) ID of the crashed process."),
+    Column("path", TEXT, "Path to the crashed process."),
+    Column("log_location", TEXT, "Location of log file."),
```

Comment:
How about `crash_path` instead. It's best to provide hints towards the type of data there-within, and we use `*path*` a lot!